package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "@snyk/code-client",
+  "version": "0.0.1",
```

Comment:
`required`: please keep version empty - it is automatically handled by semantic release.

Reply:
done